pkg/fakerp/customer_handlers.go

```diff
@@ -36,7 +36,7 @@ func (s *Server) handleDelete(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	err = dm.deleteOCPDNS(req.Context(), cs)
+	err = dm.deleteAroDns(req.Context(), cs)
```

Comment:
nit: it brings the necessary information. `dn.deleteDns` is fine too. I think there is no need to bake in names into functions names. I think it was a mistake in the first place to add `OCP`

Reply:
sure I can change this. I just did a copy-replace however this is a good point.